backend/sites/src/app/resolvers/associatedSite/associatedSite.resolver.ts

```diff
@@ -34,14 +34,17 @@ export class AssociatedSiteResolver {
   @UsePipes(new GenericValidationPipe()) // Apply generic validation pipe
   async getAssociatedSitesBySiteId(
     @Args('siteId', { type: () => String }) siteId: string,
+    @Args('pending', { type: () => Boolean, nullable: true })
+    showPending: boolean,
+    
   ) {
     this.sitesLogger.log(
       'AssociatedSiteResolver.getAssociatedSitesBySiteId() start siteID:' +
```

Comment:
please have the new parameter captured in the logs

Reply:
done